packages/familie-tidslinje/src/components/tidslinje/TimelinePeriod.tsx

```diff
@@ -156,15 +155,13 @@ const PeriodeKnapp = styled.button(
             background: #9bd0b0;
         }
     }
-`,
-);
+    ${fellesPeriodeStyle}
```

Comment:
Måtte flytte fellesPeriodeStyle nederst for at det skulle kompilere

Reply:
Mulig det hadde vært bedre om fellesPeriodeStyle hadde vært en styled div og denne heller "arvet" fra den - f.eks. 
`styled(FellesPeriodeDiv)...` ?